### PR TITLE
[zh] Sync 1.17.x change note into Chinese and fix a typo (dup space) in en version

### PR DESCRIPTION
**Pull request by wilsonwu**
Sync 1.17.x change note into Chinese (Fix a typo (dup space) in en version):
```
content\zh\news\releases\1.17.x\announcing-1.17\change-notes\index.md
```

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/news/releases/1.17.x/announcing-1.17/change-notes/index.md [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+---
+
+## 弃用通知{#deprecation-notices}
+
+以下通知说明了根据 [Istio 的弃用政策](/zh/docs/releases/feature-stages/#feature-phase-definitions)将在未来某个版本中移除的功能。
+请考虑升级您的环境以移除弃用的功能。
+
+- **弃用** 针对版本低于 1.20 的 Kubernetes 环境，
+会将 `PILOT_CERT_PROVIDER` 设置为 `kubernetes`。
+[PR #42233](https://github.com/istio/istio/pull/42233)
```
Comment (windsonsea) [inline content/zh/news/releases/1.17.x/announcing-1.17/change-notes/index.md:18]:
```suggestion
- **弃用** 针对版本低于 1.20 的 Kubernetes 环境，
  弃用了将 `PILOT_CERT_PROVIDER` 设置为 `kubernetes` 的操作行为。
  [PR #42233](https://github.com/istio/istio/pull/42233)
```

File: content/zh/news/releases/1.17.x/announcing-1.17/change-notes/index.md [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+
+- **弃用** 请使用 OpenTelemetry 提供程序替代 Lightstep 提供程序。
+[Issue #40027](https://github.com/istio/istio/issues/40027)
```
Comment (windsonsea) [inline content/zh/news/releases/1.17.x/announcing-1.17/change-notes/index.md:21]:
```suggestion
- **弃用** 弃用了 Lightstep 提供程序。请使用 OpenTelemetry 提供程序替代 Lightstep 提供程序。
  [Issue #40027](https://github.com/istio/istio/issues/40027)
```

File: content/zh/news/releases/1.17.x/announcing-1.17/change-notes/index.md [hunk 3]
```diff
@@ -0,0 +1,263 @@
...
+
+## 流量治理{#traffic-management}
+
+- **改进** `MostSpecificHostMatch` 以防止在遇到通配符时执行完整扫描主机操作。
+[Issue #41453](https://github.com/istio/istio/issues/41453)
```
Comment (windsonsea) [inline content/zh/news/releases/1.17.x/announcing-1.17/change-notes/index.md:26]:
```suggestion
- **改进** 改进了 `MostSpecificHostMatch` 以防止在遇到通配符时执行完整扫描主机操作。
  [Issue #41453](https://github.com/istio/istio/issues/41453)
```

**Review by windsonsea — COMMENTED** [review]
有 2 个普遍问题：

1. 所有 `-` 无序列表折行后，可以增加缩进，美化编辑页面。可以用一条命令解决：`npx prettier --w  content/zh/news/releases/1.17.x/announcing-1.17/change-notes/index.md`
2. `**弃用**` 这种加粗的 2 个字类似一个小标题，后面的文字是对标题的描述，所以常会增加：弃用了什么、改进了什么、新增了什么。这样读起来会有节奏感，方便获取信息。参阅 [1.16 change notes](https://github.com/istio/istio.io/blob/master/content/zh/news/releases/1.16.x/announcing-1.16/change-notes/index.md)

**Comment by wilsonwu** [comment]
> 有 2 个普遍问题：
> 
> 1. 所有 `-` 无序列表折行后，可以增加缩进，美化编辑页面。可以用一条命令解决：`npx prettier --w  content/zh/news/releases/1.17.x/announcing-1.17/change-notes/index.md`。参阅 [prettier 文档](https://prettier.io/docs/en/install.html)
> 2. `**弃用**` 这种加粗的 2 个字类似一个小标题，后面的文字是对标题的描述，所以常会增加：弃用了什么、改进了什么、新增了什么。这样读起来会有节奏感，方便获取信息。参阅 [1.16 change notes](https://github.com/istio/istio.io/blob/master/content/zh/news/releases/1.16.x/announcing-1.16/change-notes/index.md)

Got it, I will update all related problems soon.

File: content/zh/news/releases/1.17.x/announcing-1.17/change-notes/index.md (hunk 1 above)
Comment (wilsonwu) [reply content/zh/news/releases/1.17.x/announcing-1.17/change-notes/index.md]: Fixed in latest commit.

File: content/zh/news/releases/1.17.x/announcing-1.17/change-notes/index.md (hunk 2 above)
Comment (wilsonwu) [reply content/zh/news/releases/1.17.x/announcing-1.17/change-notes/index.md]: Fixed in latest commit.

File: content/zh/news/releases/1.17.x/announcing-1.17/change-notes/index.md (hunk 3 above)
Comment (wilsonwu) [reply content/zh/news/releases/1.17.x/announcing-1.17/change-notes/index.md]: Fixed in latest commit.

**Review by windsonsea — APPROVED** [review]
Thanks, LGTM

**Comment by windsonsea** [comment]
zh version is good to me now. cc @ericvn

**Review by ericvn — APPROVED** [review]
Thanks.

**Pull request merged**